build_msvc/common.init.vcxproj

```diff
@@ -6,7 +6,12 @@
     <VCProjectVersion>16.0</VCProjectVersion>
     <VcpkgTriplet Condition="'$(Platform)'=='Win32'">x86-windows-static</VcpkgTriplet>
     <VcpkgTriplet Condition="'$(Platform)'=='x64'">x64-windows-static</VcpkgTriplet>
-</PropertyGroup>
+    <QtBaseDir>C:\Qt5.9.7_ssl_x64_static_vs2017</QtBaseDir>
+    <QtIncludeDir>$(QtBaseDir)\include</QtIncludeDir>
+    <QtLibraryDir>$(QtBaseDir)\lib</QtLibraryDir>
+    <QtToolsDir>$(QtBaseDir)\bin</QtToolsDir>
+    <QtPluginsLibraryDir>$(QtBaseDir)\plugins</QtPluginsLibraryDir>
```

Comment:
If you remove what is repeated in the `bitcoin-qt` project, you don't have to declare anything more than `QtBaseDir`